subsys/nonsecure/Kconfig

```diff
@@ -0,0 +1,314 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if BUILD_WITH_TFM || IS_SPM
+menu "Trusted Execution Non-Secure"
+menu "Peripheral Secure mapping"
+
+config NRF_FPU_SECURE
+	bool "FPU configured as secure peripheral"
+	depends on CPU_HAS_FPU
```

Comment:
General comment.

All settings in this file have no help text.

Are we sure all users fully understand what it implies to configure a peripheral as secure ?

Could we have just a brief description and maybe a reference to more information.
Remember the help text is rendered into our docs, like this:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/CONFIG_BUILD_WITH_TFM.html#std-kconfig-CONFIG_BUILD_WITH_TFM